packages/device-api/package.json

```
@@ -0,0 +1,21 @@
+{
+  "private": true,
+  "version": "1.0.0",
+  "description": "RPC-style calls with Zod validations + type safety",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/duckduckgo/duckduckgo-autofill.git"
+  },
+  "keywords": [
+    "duckduckgo",
+    "zod",
+    "rpc"
+  ],
+  "author": "",
+  "license": "ISC",
```

FYI, we always use Apache-2.0, unless we need something else explicitly.

```suggestion
  "license": "Apache-2.0",
```